tests/support/fixtures.ts

```diff
@@ -229,15 +230,35 @@ export async function createSeedFixture() {
     "Git repository for source browsing tests",
     "--announce",
   ]);
+  await createIssue(
```

Comment:
I know this is still a draft, but I was wondering whether we shouldn't already start a new fixture for the cob tests so it doesn't end up growing into one huge fixture?